src/main/java/net/atomique/ksar/XMLConfig.java

```diff
@@ -28,71 +30,60 @@
 import javax.xml.parsers.SAXParserFactory;
 
 public class XMLConfig extends DefaultHandler {
+  private static final String KSAR_DTD_PREFIX = "-//NET/ATOMIQUE/KSAR/";
 
   private static final Logger log = LoggerFactory.getLogger(XMLConfig.class);
 
-  XMLConfig(String filename) {
-    load_config(filename);
-
-  }
-
-  XMLConfig(InputStream is) {
-    load_config(is);
-  }
-
   public InputSource resolveEntity(String publicId, String systemId)
-      throws SAXException, IOException {
+      throws IOException {
 
-    InputSource inputSource = null;
-    try {
-      String dtdFile = systemId.substring(systemId.lastIndexOf("/"));
-      InputStream inputStream = EntityResolver.class.getResourceAsStream(dtdFile);
-      inputSource = new InputSource(inputStream);
-    } catch (Exception e) {
-      // No action; just let the null InputSource pass through
+    if (publicId == null || !publicId.startsWith(KSAR_DTD_PREFIX)) {
+      return null;
     }
 
-    // If nothing found, null is returned, for normal processing
-    return inputSource;
+    String dtdFile = publicId.substring(KSAR_DTD_PREFIX.length() - 1);
```

Comment:
because of String change without trainling / 
String dtdFile = publicId.substring(KSAR_DTD_PREFIX.length());